src/untangled/server/impl/components/access_token_handler.clj

```diff
@@ -96,9 +96,8 @@
               token (get-token request)]
           (if-not (valid-token? token merged-options)
             (let [{:keys [invalid-token-handler]} merged-options]
-              (if-let [_ok-anyway? ((or invalid-token-handler (constantly false)) request)]
-                (handler request)
-                {:status 401}))
+              ((or invalid-token-handler (constantly false)) request)
+              (handler request))
```

Comment:
All it does now is pass through the handler chain, when we do not have a valid token, with user will be associated with the request.    This is what we discussed with Tony.  This puts control into the actual code that cares about authorization to look for the user and do the right things based on that.